### DISCUSSION TITLE
请问一下通过脚手架安装的nonebot怎么更新

### DISCUSSION THREAD

**Discussion by forchannot** [nonebot/discussions#149] [问答] [answered]
如题，通过脚手架安装的nonebot要如何更新，输入nb好像没找到更新命令，还是直接nb create？

**Comment by yanyongyu** [comment, marked as answer]
`pip install --upgrade nonebot2`

**Reply by forchannot** [reply]
> `pip install --upgrade nonebot2`

也就是说nonebot本质只是个python的包？直接pip升级就好了？

**Reply by yanyongyu** [reply]
是的:joy:

**Reply by forchannot** [reply]
> 是的😂

好的，感谢

**Reply by xcu2024** [reply]
？
